fake-manage-recalls-api/stubs/__files/get-recall.json

```diff
@@ -10,6 +10,7 @@
   "agreeWithRecall": "YES",
   "agreeWithRecallDetail": "Reasons...",
   "previousConvictionMainName": "Walter Holt",
+  "licenceNameCategory": "FIRST_LAST",
```

Comment:
This won't pass PACT until its being returned by the API, can you move it to the next PR?

Reply:
oops, removed